src/Config/Database/DbConfig.php

```diff
@@ -0,0 +1,285 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Config\Database;
+
+use Magento\MagentoCloud\Config\ConfigMerger;
+use Magento\MagentoCloud\Config\Stage\DeployInterface;
+use Magento\MagentoCloud\DB\Data\ConnectionInterface;
+use Magento\MagentoCloud\DB\Data\RelationshipConnectionFactory;
+
+/**
+ * Returns merged database configuration from the .magento.env.yaml file
+ * and environment variable MAGENTO_CLOUD_RELATIONSHIPS.
```

Comment:
environment variable MAGENTO_CLOUD_RELATIONSHIPS - this isn't always true as the name of this variable is configurable.

Reply:
name was removed from description